packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -0,0 +1,178 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
+
+enum MobileAdEvent {
+  loaded,
+  failedToLoad,
+  clicked,
+  impression,
+  opened,
+  leftApplication,
+  closed,
+}
+
+typedef void MobileAdListener(MobileAdEvent event);
+
+class MobileAdTargetingInfo {
+  const MobileAdTargetingInfo({
+    this.keywords,
+    this.contentUrl,
+    this.birthday,
+    this.gender,
+    this.designedForFamilies,
+    this.childDirected,
+    this.testDevices,
+    this.requestAgent,
+  });
+
+  final List<String> keywords;
+  final String contentUrl;
+  final DateTime birthday;
+  final String gender;
```

Comment:
It looks like gender is just translated into an enum value on the native side. Should a similar enum for gender be exposed here?